util/log.go

```diff
@@ -35,8 +35,11 @@ func InitLog(logLevel string, logPath string) error {
 		AddSyslogHook()
 	}
 
+	//nolint:gocritic
```

Comment:
> ifElseChain: rewrite if-else to switch statement (gocritic)

hmm I'm not sure how switch-case can be used here - I always thought that you can apply it for the same variable  which seams not be a case here. I decided to selectively disable linter here